### DISCUSSION TITLE
There was a problem with \/return\/policy : cannot be null

### DISCUSSION THREAD

**Discussion by scuculeac-plenty** [etsy/open-api#904] [Q&A]
Hi. We have an issue that states "There was a problem with /return/policy : cannot be null" even tho we don't send "return policy id" parameter at all. (nether on create/update listing)

Attached is what we send to etsy, and the response is 
 Client error: `PATCH https://openapi.etsy.com/v3/application/shops/xxxxx/listings/yyyyy?language=de` resulted in a `400 Bad Request` response:
{"error":"There was a problem with \/return\/policy : cannot be null"}

Please give us more information's about what we do wrong in this case?

Best Regards,
Stefan
 
![Screenshot 2023-03-09 at 11 59 50](https://user-images.githubusercontent.com/82037958/224004019-d03f18e6-c002-4040-810b-ee655e6e9002.png)

**Comment by PMStanley** [comment]
Sounds like you might be running foul of the breaking change they introduced in Jan: https://github.com/etsy/open-api/discussions/761

Check if the listing already has a returns policy, if not you will need to set one at the time of creation/update

**Reply by scuculeac-plenty** [reply]
Hi @PMStanley, this client has setup an return policy and is not outside of EU, so we shouldn't get that error based on https://github.com/etsy/open-api/discussions/761

**Reply by PMStanley** [reply]
You could try including the existing returns policy in the API request. Also worth emailing Etsy directly to see if there's a technical issue.
